docs/src/main/asciidoc/datasource.adoc

```diff
@@ -85,7 +85,17 @@ quarkus.datasource.jdbc.max-size=16
 <1> This configuration value is only required if there is more than one database extension on the classpath.
 
 If only one viable extension is available, Quarkus assumes this is the correct one.
-If a driver has been added to the test scope, Quarkus automatically includes the driver in testing.
+When a driver has been added to the test scope, Quarkus automatically includes the driver in testing.
```

Comment:
Make active. If the developer always does this action. Otherwise, leave as-is and close this comment.

```suggestion
When you add a driver to the test scope, Quarkus automatically includes the specified driver in testing.
```